packages/camel-model/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
```

Comment:
we might want to revisit these options, especially on "jsx" where we shouldn't need react & jsx support here.